lms/static/scripts/frontend_apps/config.js

```diff
@@ -41,6 +41,14 @@ import { createContext } from 'preact';
  * @prop {Object} submissionParams
  */
 
+/**
+ * Data needed to display the VitalSource book viewer.
+ *
+ * @typedef VitalsourceConfig
+ * @prop {string} launchUrl
+ * @prop {Object.<string,string>} launchParams
```

Comment:
Or link to docs here? Now wondering if we should have a separate typedef for this...

See https://developer.vitalsource.com/hc/en-us/articles/206156238-General-LTI-Usage

Reply:
The intent here is that the frontend doesn't need to know about what the form parameters mean, it just renders a form given whatever parameters the backend supplies and submits it. The supported parameters are documented in the backend code that generates the them.